hasher-matcher-actioner/webapp/.watchmanconfig

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Couldn't run tests without this file. Error that was showing led to the creation of this empty .watchmanconfig file.